plugins/flytekit-vscode/flytekitplugins/vscode/__init__.py

```diff
@@ -0,0 +1,13 @@
+"""
+.. currentmodule:: flytekitplugins.vscode
+
+This package contains things that are useful when extending Flytekit.
```

Comment:
Is this sentence from the plugin template?

Reply:
This sentence is in every plugin so I assume it's a kind of a template.